test/test_torch.py

```diff
@@ -9322,6 +9322,10 @@ def test_kthvalue(self, device, dtype):
             self.assertEqual(res1val[:, :], res2val[:, :, k - 1], atol=0, rtol=0)
             self.assertEqual(res1ind[:, :], res2ind[:, :, k - 1], atol=0, rtol=0)
 
+        # Test scalar input
```

Comment:
Expand this comment with the issue it's fixing and briefly explain why the test is appropriate.